release-notes/6.0/releases.json

```diff
@@ -154,13 +154,13 @@
           },
           {
             "name": "dotnet-sdk-osx-arm64.pkg",
-            "rid": "arm64",
+            "rid": "osx-x64",
```

Comment:
osx-x64 or osx-arm64 @rbhanda?

Reply:
osx-arm64

Reply:
That makes more sense indeed, but maybe you should then also change it for all the other osx-arm64 packages 😉. I changed it from arm64 to osx-x64 because it was inconsistent between the sdk field the the first item in the sdks field, and all other osx-arm64 files in this releases.json file were labeled as osx-x64.

Reply:
Ok, I'll create a PR to fix that